sympy/solvers/solveset.py

```diff
@@ -1210,12 +1210,13 @@ def _solve_modular(f, symbol, domain):
         The variable in the equation to be solved.
 
     domain : Set
-        A set over which the equation is solved.
+        A set over which the equation is solved. It has to be a subset of
+        Integers.
 
     Returns
     =======
 
-    A set of solutions satisfying the given modular equation.
+    A set of integral solutions satisfying the given modular equation.
```

Comment:
What's `integral`?

Reply:
Ohh it should be integer. I changed it just now.